licenses/SteamCMD

```diff
@@ -0,0 +1,897 @@
+  STEAM® SUBSCRIBER AGREEMENT
```

Comment:
Why are you naming it SteamCMD when it's called 'steam subscriber agreement'?

Reply:
Because my local Steam client shows a different license, so I named it SteamCMD, to avoid confusion with Steam, if it comes some day to portage. Should I still rename it?

Reply:
So they have multiple subscriber agreements?

Reply:
I revert my previous text ;-) It seems, they are the same, so I change the license name to `Steam`.